app/routes/vaccinate.js

```diff
@@ -124,6 +126,66 @@ module.exports = router => {
     }
   })
 
+  router.get('/vaccinate/patient', (req, res) => {
+
+    const data = req.session.data
+    const showError = data.showError
+    const nhsNumberKnown = data.nhsNumberKnown
+    const nhsNumber = String(data.nhsNumber).replaceAll(' ', '')
+
+    let nhsNumberKnownError, nhsNumberError
+    let errorList = []
+
+    if (req.query.showError == 'yes') {
+
+      if (!nhsNumberKnown) {
+        nhsNumberKnownError = {
+          text: "Select yes if you have the patient's NHS number",
+          href: "#nhs-number-known-1"
+        }
+        errorList.push(nhsNumberKnownError)
+      }
+
+      if (nhsNumberKnown === 'yes') {
+
+        if (nhsNumber == '') {
+          nhsNumberError = {
+            text: "Enter NHS number",
+            href: "#nhs-number"
+          }
+          errorList.push(nhsNumberError)
+        } else if (nhsNumber.match(/[^\d]/)) {
+          nhsNumberError = {
+            text: "NHS number must only contain digits",
```

Comment:
Suggestion from @Anna-Sutton:
```suggestion
            text: "Enter a 10-digit number",
```